src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.CoreCLR.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is this the right head? It's different from what we use in other files.

// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.